build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -6,6 +6,32 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="xunit" Version="$(XUnitVersion)" IsImplicitlyDefined="true" PrivateAssets="all"/>
```

Comment:
📝 `$(xunitVersion)` is how it's declared. Some of the others may have casing differences as well.